packages/renderer/src/lib/style/IconsStyle.svelte

```diff
@@ -19,7 +19,7 @@ function createStyleSheet(): HTMLStyleElement {
 }
 
 export function toUrl(location: string) {
-  return `url('file://${location.replace(/'/g, '%27')}')`;
+  return `url('file://${location.replace(/\\/g, '/').replace(/'/g, '%27')}')`;
```

Comment:
should the location be replaced on the server side ?

Reply:
there: https://github.com/containers/podman-desktop/blob/main/packages/main/src/plugin/icon-registry.ts#L85-L101

you changed to use join method when checking some file but then probably location should be set properly here ?

Reply:
the fact is that the path is not wrong and if we change it in the server side it could create other issues if we need to work with it in future (comparing, equals ...) . BTW i didn't have a strong opinion on it so i moved the change on the server side. Only for Windows as it only seems affecting it

Reply:
@lstocchi maybe we can add on server side a new field browserURL and just use that field on the client side ?

Reply:
so we provide both links, one for the 'server', one for the 'client' ?